### DISCUSSION TITLE
Open a Modal from a different component

### DISCUSSION THREAD

**Discussion by bwosborne2** [wire-elements/modal#216] [Q&A] [answered]
How to you open a Modal from a different Livewire Component ( not blade) ?

I have a component named Devices and, in certain situations, I want to open OwnerModal

**Comment by PhiloNL** [comment]
You can use javascript and the Livewire object:

```
<!-- Outside of any Livewire component -->
<button onclick="Livewire.emit('openModal', 'edit-user')">Edit User</button>

<!-- Inside existing Livewire component -->
<button wire:click="$emit('openModal', 'edit-user')">Edit User</button>

<!-- Taking namespace into account for component Admin/Actions/EditUser -->
<button wire:click="$emit('openModal', 'admin.actions.edit-user')">Edit User</button>
```

**Reply by bwosborne2** [reply]
I am trying to show the OwnerModal by calling it within the mount() method of the Devices component. No button is involved.

**Reply by PhiloNL** [reply]
It's not possible to call another modal from the mount() method due to how Livewire works.

**Reply by bwosborne2** [reply]
OK, Thank you.

**Comment by bwosborne2** [comment, marked as answer]
Answered that I cannot do what I was attempting.
